microprofile.jdt/org.eclipse.lsp4mp.jdt.core/src/main/java/org/eclipse/lsp4mp/jdt/internal/faulttolerance/java/MicroProfileFaultToleranceASTValidator.java

```diff
@@ -121,9 +132,14 @@ public boolean visit(TypeDeclaration type) {
 							validateAsynchronousAnnotation(node, (MarkerAnnotation) modifier);
 						}
 					} catch (JavaModelException e) {
-						LOGGER.log(Level.WARNING, "An exception occurred when attempting to validate the annotation");
+						e.printStackTrace();

```

Comment:
Avoid just leaving these as `e.printStrackTrace()` . When you catch an exception, just ask yourself whether you expect it to ever fail, and if so, whether you can recover, or simply reporting the error is the best that can be done.

In this case, it's like you just want to report the error using `LOGGER.log(..) as it was before on the left.